astronomer/starship/services/astro_client.py

```diff
@@ -90,7 +96,7 @@ def get_deployment_url(deployment, token):
 
 
 def set_environment_variables(
-    deployment: str, token: str, remote_vars: dict[dict[str, Any]]
+    deployment: str, token: str, remote_vars: Dict[Dict[str, Any]]
```

Comment:
apparently this was fine in >=py3.9
`from __future__ import annotations` would've made it fine in py3.7+3.8
this `from typing import Dict` makes it work with py3.5+3.6